packages/loader/container-definitions/src/runtime.ts

```diff
@@ -120,6 +121,7 @@ export interface IContainerContext extends IDisposable {
     readonly submitFn: (type: MessageType, contents: any, batch: boolean, appData?: any) => number;
     readonly submitSignalFn: (contents: any) => void;
     readonly closeFn: (error?: ICriticalContainerError) => void;
+    readonly loadContainerCopyFn: (additionalHeaders: IRequestHeader) => Promise<IContainer>,
```

Comment:
Let's have explicit parameters here to constrain what options can be provided.  With `IRequestHeader` it's not obvious what options might be passed, and what the implementation might be looking for.